print/ghostscript/Portfile

```diff
@@ -162,6 +197,14 @@ variant cups description {Enable CUPS driver} {
     configure.args-replace  --disable-cups --enable-cups
 }
 
+variant ghostpdl description {Install full GhostPDL} {
+    distfiles-replace       ${distname}.tar.gz:source \
+                            ${ghostpdl_distname}.tar.gz:source
+    extract.only-replace    ${distname}.tar.gz \
+                            ${ghostpdl_distname}.tar.gz
+    configure.args-append   --enable-gpdl
+}
+
```

Comment:
Why make this a variant? Why not always do this?

Reply:
Sure, that would be fine.  Thoughts on changing the port name to ghostpdl?

Reply:
After thinking about this some more, I think it makes sense to switch to the GhostPDL source distribution but make the gpcl6/gxps/gpdl binaries an optional variant.  These tools are used for HP PCL and Microsoft XPS files which are somewhat obscure and probably not that useful for most users.

Reply:
We definitely don't want to change the port name because many port depends on "ghostscript" and those would all have to change. I'm not knowledgeable enough about ghostscript/ghostpdl but may a sub-port is warranted for it?

Reply:
Okay, that makes sense.  Based on my understanding of sub-ports I think a variant is more suitable.